src/content/blogs/platform_v2.md

```diff
@@ -183,4 +203,8 @@ On the tooling side of Popgetter, we are planning on building a number of differ
 want from Popgetter, we are planning to build out a terminal user interface and web interface. We also think there is great utility in making Popgetter available in the data science and web tooling
 contexts. To enable those use cases, we will be developing Python and JavaScript interfaces for the Popgetter library.
 
-####
+#### Routing best practices: a guide
```

Comment:
I'd maybe leave this one out. If somebody clicks through to the guide right now, they'll just see a confusing WIP thing. I'd vote to just not mention it until it's minimally readable

Reply:
Sounds good, I'll go ahead and remove this section.